src/components/cards/constants.ts

```diff
@@ -0,0 +1,184 @@
+import { globalColors } from '@/design-system';
+
+type LearnCategory =
+  | 'Essentials'
```

Comment:
prob worth i18n this stuff, id imagine we'd do this at the source but for now we should make sure we're covered